test/gc-stress/config-gc-bench-local.json

```diff
@@ -1,9 +1,11 @@
 {
     "distSpecVersion": "1.1.0-dev",
     "storage": {
-        "rootDirectory": "/tmp/zot",
+        "rootDirectory": "/tmp/zot/local",
         "gc": true,
-        "gcDelay": "10s",
+        "gcReferrers": true,
```

Comment:
@peusebiu can you add/copy the same (both local and s3) test with gcReferrers=false also?